### DISCUSSION TITLE
Different types of dispatchers on the same Elsa server

### DISCUSSION THREAD

**Discussion by tsoG2000** [elsa-workflows/elsa-core#1620] [Q&A] [answered]
Is it possible to configure Elsa (and how) to use different types of dispatchers, default Rebus with inMemoryTransport (if it runs faster) for some types of workflows and Hangfire dispatcher for other?

**Comment by sfmskywalker** [comment, marked as answer]
There's no such way at the moment, but I think it would be great to have. Please feel free to submit a feature request.
